addons/mail/static/src/components/top_bar/top_bar.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.TopBar" owl="1">
+        <div class="o_TopBar">
+            <span class="o_TopBar_member">
+                <t t-esc="thread_name"/>
+                (<t t-esc="members_count"/> members)
+            </span>
```

Comment:
All of this should be translatable string, because whole format is locale dependent.